Bucketeer/Sources/Internal/Event/EventInteractor.swift

```diff
@@ -168,6 +168,19 @@ final class EventInteractorImpl: EventInteractor {
     }
 
     func trackFetchEvaluationsFailure(featureTag: String, error: BKTError) throws {
+
+        if case .forbidden = error {
+            logger?.warn(message: "An forbidden error occurred. Please check your API Key.")
```

Comment:
Can you merge the warning and error log?
What will happen if the register event API request fails?  Does it also log the errors?

Reply:
unfortunately, the logger interface doesn't support passing the error as an input parameter like the Android SDK. I don't want to introduce breaking changes on the logger.

Reply:
> What will happen if the register event API request fails? Does it also log the errors?

we did nothing when it failed. The events still on the database and waiting for the next synchronization

Reply:
I updated it, we will not generate a log when the register event fails.
https://github.com/bucketeer-io/ios-client-sdk/pull/95/commits/fbcf080ec36aba557d2e2b35edb71ee35f15c17e